matter/wifi/ethernetif.cpp

```diff
@@ -185,7 +186,7 @@ static err_t low_level_output(struct netif *netif, struct pbuf *p)
   result = SL_STATUS_FAIL;
 
   //EFR32_LOG ("WF200: Out %d", (int)framelength);
-  while ((result != SL_STATUS_OK) && (i++ < INT_VALUE_10)) {
+  while ((result != SL_STATUS_OK) && (i++ < 10)) {
```

Comment:
There is no explanation for the 10. why do try 10 times

Reply:
Junior Even I am not aware of this why ten frames are sent in one go.
I tried to find out the explanation for it but could not able to get much info.

I will this pending for now.

Reply:
I will keep this pending for now.